protocols/bsbport/bsbport_ecmd.c

```diff
@@ -33,6 +33,12 @@
 #include "protocols/bsbport/bsbport_tx.h"
 #include "protocols/ecmd/ecmd-base.h"
 
+#ifdef DEBUG_BSBPORT_ECMD
+#define BSBPORT_DEBUG(s, args...) debug_printf("BSB " s "\n", ## args);
```

Comment:
Why are we using these old-fashioned conctruct? SInce we compile at least with C99 why not use variadic macros?